h5p-bildetema-topic-list/src/components/Word/Word.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import * as React from "react";
+import Word from "./Word";
+import {Word as WordType} from '../../../../common/types/types'
+
+export default {
+  title: "Word",
+  component: Word,
+} as ComponentMeta<typeof Word>;
+
+const Template: ComponentStory<typeof Word> = (args) => (
+  // eslint-disable-next-line react/jsx-props-no-spreading
+  <Word {...args}/>
+);
+
+export const Default = Template.bind({});
+Default.args = {
+  clickHandler(word : WordType) {
+    console.log(word)
+  },
+  word: {
+    id: "V0889",
+    label: "Elg",
+    images: ["http://oddestemmen-camp.no/images/phocagallery/Tierbilder/thumbs/phoca_thumb_l__mg_2903.jpg"]
```

Comment:
This is really nitpicky, but because this codebase is open source, I don't think we should use images we don't have the license for. Instead, we can use images from https://unsplash.com and other "do as you want" sources.

```suggestion
    images: ["https://images.unsplash.com/photo-1549471013-3364d7220b75?crop=entropy&cs=tinysrgb&fm=jpg&ixlib=rb-1.2.1&q=80&raw_url=true&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2340"]
```